### PR TITLE
Create packages.md

### PR DESCRIPTION
**Pull request by adydychk**

### CODE REVIEW

File: docs/packages.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
+# Packages description needed for building Intel® Media SDK
+The prupose of this guide is to clarify why Intel® Media SDK need specific packages and the dependencies of packages.
+
+### Packages from MSS
```
Comment (azhogov) [inline docs/packages.md]: Is MSS ok?

File: docs/packages.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+```
+
+LibDRM  
+[It](https://01.org/linuxgraphics/community/libdrm) is the cross-driver middleware which allows user-space applications to communicate with the Kernel by the means of the DRI protocol.  
+**LibDRM is dependency of LibVA.**
+```bash
+libdrm-xxx-xxx.el7.centos.x86_64.rpm
+libdrm-devel-xxx-xxx.el7.centos.x86_64.rpm
+```
+LibDRM may be from OS repositories but it is not recommended!
```
Comment (azhogov) [inline docs/packages.md]: Why?

File: docs/packages.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+sudo yum groupinstall "Development Tools"
+sudo yum install cmake git
+sudo yum install libX11 libXext libXfixes libGL libGL-devel libX11-devel 
+```
+Analyze these packages in more detail:  
+`Development Tools` are mainly needed for compilers (such as gcc, g++) and other things which are useful for building the product.  
+
+`cmake` and `git` are needed only on pre-build stage.  
+
+`libX11 libXext libXfixes libGL libGL-devel libX11-devel` all these packages needed while building the product.
```
Comment (azhogov) [inline docs/packages.md]: are needed

File: docs/packages.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+
+### System`s packages
+Here is a list of recommended packages
+```bash
+sudo yum groupinstall "Development Tools"
+sudo yum install cmake git
+sudo yum install libX11 libXext libXfixes libGL libGL-devel libX11-devel 
+```
+Analyze these packages in more detail:
```
Comment (azhogov) [inline docs/packages.md]: Purpose of the packages

File: docs/packages.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+Intel® Media SDK can be built without opencl packages but you will get the product without some features.  
+How to build without opencl? - Simply don`t install these packages on your system. Build scripts will automatically generate needed make files.
+
+LibVA
+```bash
+libva-xxx-xxx.el7.centos.x86_64.rpm
+libva-devel-xxx-xxx.el7.centos.x86_64.rpm   # Needed for headers in /usr/include/va
+```
+
+LibDRM
```
Comment (azhogov) [inline docs/packages.md]: link should be here. "It" is not good as link

File: docs/packages.md (hunk 2 above)
Comment (adydychk) [reply docs/packages.md]: Why it is not recommended?

File: docs/packages.md [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+Analyze these packages in more detail:  
+`Development Tools` are mainly needed for compilers (such as gcc, g++) and other things which are useful for building the product.  
+
+`cmake` and `git` are needed only on pre-build stage.  
+
+`libX11 libXext libXfixes libGL libGL-devel libX11-devel` all these packages needed while building the product.
+
+
+### Tip
+In case if you have troubles with "recommended list of needed packages" you can try to use this a bit bigger list:
```
Comment (azhogov) [inline docs/packages.md]: What kind of troubles do you mean?

Comment (adydychk) [reply docs/packages.md]: For example if some packages will be NOT resolved as dependency (as a result they will be not automatically downloaded). In this case here is (almost) full list of packages :)

File: docs/packages.md (hunk 2 above)
Comment (azhogov) [reply docs/packages.md]: Yes :)

File: docs/packages.md (hunk 6 above)
Comment (azhogov) [reply docs/packages.md]: Just write there are full list of required packages...

**Pull request merged**